src/WindowsAzureMessaging/WindowsAzureMessaging/Vendor/Reachability/MS_Reachability.m

```diff
@@ -0,0 +1,235 @@
+/*
```

Comment:
Will this conflict with the App Center SDK?

Reply:
Tested! No conflicts